corehq/ex-submodules/pillowtop/processors/form.py

```diff
@@ -39,8 +48,12 @@ def process_change(self, change):
             # the same effect, an app having has_submissions set to True.
             mark_has_submission(domain, build_id)
 
+        try:
+            received_on = string_to_utc_datetime(doc.get('received_on'))
+        except ValueError:
+            return
```

Comment:
this is already happening in the `mark_latest_submission` helper. I added it here as well, but thinking I can get rid of it in the helper now